api/grpc/proto/user.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package user;
+
+option go_package = "github.com/ezex-io/ezex-users/api/grpc/proto";
+
+// UsersService provides operations for managing users.
+service UsersService {
+  // SaveSecurityImage saves a new security image for a user.
+  rpc SaveSecurityImage(SaveSecurityImageRequest) returns (SaveSecurityImageResponse) {}
+
+  // GetSecurityImage retrieves a security image by its ID.
+  rpc GetSecurityImage(GetSecurityImageRequest) returns (GetSecurityImageResponse) {}
+}
+
+// SaveSecurityImageRequest represents a request to save a security image.
+message SaveSecurityImageRequest {
+  // The unique identifier of the user who owns the security image.
+  string user_id = 1;
+  // The binary data of the security image in bytes.
+  bytes image_data = 2;
```

Comment:
Security image contains two items:

```
// Image selected by the user to confirm authentic login page.
string security_image = 2;

// Phrase set by the user to confirm authentic login page.
string security_phrase = 3;
```

It doesn't make sense to store the image binary, as it would be repeated and increase the size of the database.  
Storing image names like `moon.png` is a much better way to handle the security images phase.